atlp-rwanda/atlp-devpulse-fn
# Fix signup

### PR Description

This PR fixes the issue of signup whereby bio had been added to the schema but not validated

### Description of tasks that were expected to be completed

- As a user, I should be able to signup

### How can this been tested?
- Clone repo
- Checkout branch `fix-signup`
- Make sure that you set the environment variables referring to `.env.example`
- Run `npm install`, then `npm run dev` to check if the app is running successfully
- Use apolo server and interact with doc resolvers


### Track PR (issue number & link)
N/A

### Screenshots (If appropriate especially on the frontend)
N/A
